src/OpeningHours.php

```diff
@@ -211,6 +211,26 @@ public function forWeekCombined(): array
         return $equalDays;
     }
 
+    public function concatnatedDays(): array
+    {
+        $concatnatedDays = [];
+        $allOpeningHours = $this->openingHours;
+        foreach ($allOpeningHours as $day => $value) {
+            $previousDay = end($concatnatedDays);
+            if ($previousDay && (string) $previousDay['opening_hours'] === (string) $value) {
+                $key = array_key_last($concatnatedDays);
```

Comment:
`array_key_last` is not possible as `opening-hours` minimum compatible version is PHP 7.0.

Reply:
As i'm using end() i can use the key() method as well 👍